clients/cobol-lsp-vscode-extension/src/__tests__/services/ProcessorGroups.spec.ts

```diff
@@ -22,13 +22,14 @@ import {
   loadProcessorGroupDialectConfig,
   loadProcessorGroupSqlBackendConfig,
 } from "../../services/ProcessorGroups";
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 import { globSync } from "glob";
```

Comment:
why is it needed?

Reply:
It's because ESLint requires a variable to be accessed (something read from it) anywhere in the code. We use it just for mocking, so we just write into it, but never read it.

Reply:
I see, the variable is const/readonly, and eslint does not see through the `as any`?

Reply:
I don't think it's because of the `as any`. It's because write-only variables are not considered as used - https://eslint.org/docs/latest/rules/no-unused-vars and `globSync` is write only in this module.
It's a limitation of ESlint that it doesn't recognise it's a mock of an import and it might be used in other module.